ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<p>It will redirect, but better to go directly to new doc set</p><p> </p><p>s/ai-platform-unified/vertex-ai/</p><p> </p><p>This is a global comment, please do this for all of the links into the docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I reviewed all the links, and I think this is the only one I needed to change. I'd appreciate a double-check though!

Reply:
I agree, I don't see other links with the old url.